README.md

```diff
@@ -3,6 +3,7 @@
 A curated list of awesome Java frameworks, libraries and software.
 
 - [Awesome Java](#awesome-java)
+    - [ASCII/Console](#asciiconsole)
```

Comment:
I don't think the category is fine-grained enough. ASCII art is quite different from, e.g. interactive IO.

Also we already have a category called **command-line argument parsing** which should be the role model for this change.